core/src/main/java/io/confluent/kafka/schemaregistry/rest/SchemaRegistryConfig.java

```diff
@@ -362,6 +368,10 @@ public class SchemaRegistryConfig extends RestConfig {
   protected static final String INTER_INSTANCE_HEADERS_WHITELIST_DOC
       = "A list of ``http`` headers to forward from follower to leader, "
       + "in addition to ``Content-Type``, ``Accept``, ``Authorization``.";
+  protected static final String INTER_INSTANCE_LISTENER_NAME_DOC
+      = "Name of listener used for communication between schema registry instances. If this "
+      + "is unset, the listener name is defined by inter.instance.protocol. If both properties "
+      + "are set at the same time, inter.instance.listener.name takes precedence.";
```

Comment:
Should the DOC for inter.instance.protocol mention that inter.instance.listener.name overrides inter.instance.protocol?
Also should we mention here that this property takes precedence over host.name? And similar to the first comment, should the DOC for host.name also mention this behaviour?